syntax = "proto3";
package spacepi.messaging;
option java_package = "com.ffaero.spacepi.messaging";

import "spacepi/messaging/MessageID.proto";

message HelloMessage {
    option (MessageID) = 0x10;

    string serverID = 1;
}
